lib/protocol/src/openctx-protocol.schema.json

```diff
@@ -105,31 +114,58 @@
       "properties": {},
       "tsType": "Record<string, never>"
     },
+    "MentionSelector": {
+      "description": "List of regex patterns matching the mention text for which the provider can return mentions.",
```

Comment:
Describe more what this means, basically what you just told me: This is about supporting a 2nd way of the user selecting a mention. After typing @, they can select the mention kind OR they can type in a full URL (or something else I guess). The second way lets them bypass choosing the mention kind.

Reply:
done